hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/TestObjectBasedReadOnlyProcessor.java

```diff
@@ -0,0 +1,22 @@
+package com.hazelcast.map.impl.nearcache;
+
+import com.hazelcast.core.ReadOnly;
+import com.hazelcast.map.EntryProcessor;
+import com.hazelcast.query.SampleTestObjects;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+
+public class TestObjectBasedReadOnlyProcessor implements EntryProcessor<Integer, SampleTestObjects.Employee, Boolean>, ReadOnly {
```

Comment:
I did not understand why we need a Employee based entry processor. Can't you just use Predicates.alwaysTrue when you need a Predicate? I mean, can't we simply use IMap<Integer,Integer> for all the tests you added?